_sass/_base.scss

```
@@ -0,0 +1,141 @@
+* {
+  box-sizing: border-box;
+
+  &::after,
+  &::before {
+    box-sizing: border-box;
+  }
+}
+
+body {
+  margin: 0;
+  color: var(--color-text-body);
+  font-size: 112.5%; // 18px
+  font-family: $font-body;
+  line-height: 1.5;
+  background-color: #fff;
+}
+
+a {
+  position: relative;
+
+  &:link {
+    color: var(--color-link);
+  }
+
+  &:link,
+  &:visited {
+    text-decoration: underline;
+  }
```

@jsnmrs do you think it's ok (accessibility wise) to style :visited the same color as the link color?

```suggestion
  &:link {
    color: var(--color-link);
    text-decoration: underline;
  }
```

---

@katydecorah Yes. It's a helpful visual cue, but gets knocked over by a cache clear or private browser window. Screen readers will announce a visited link, even when the visual treatment is identical to an unvisited link.